### PR TITLE
[azsystemevents] Updating to the latest commit: fb64eaa0dfc7ee8abc8e10369495729d5b3c1cc5

### PR DESCRIPTION
**Pull request by richardpark-msft**
APIView here: https://apiview.dev/Assemblies/Review/db71641e728c4b7ba62abe3efde3cf7e/d02ef49dd66b4636b0b2bc880af74830?diffRevisionId=327a14ddd4774fdfa546875fe7487e27&doc=False&diffOnly=True

### CODE REVIEW

File: sdk/messaging/eventgrid/azsystemevents/CHANGELOG.md [hunk 1]
```diff
@@ -1,14 +1,19 @@
 # Release History
 
-## 0.4.4 (Unreleased)
+## 0.4.4 (2024-11-19)
```
Comment (chlowell) [inline sdk/messaging/eventgrid/azsystemevents/CHANGELOG.md:3]: Shouldn't this be 0.5.0? There are breaking changes

**Review by chlowell — APPROVED** [review]

File: sdk/messaging/eventgrid/azsystemevents/CHANGELOG.md (hunk 1 above)
Comment (richardpark-msft) [reply sdk/messaging/eventgrid/azsystemevents/CHANGELOG.md:3]: Good point, I'll update it.

**Comment by richardpark-msft** [comment]
@microsoft-github-policy-service rerun

**Pull request merged**